public_suffix_list.dat

```diff
@@ -12389,4 +12389,21 @@ za.org
 // Submitted by Olli Vanhoja <olli@zeit.co>
 now.sh
 
+// MyDNS.JP ： https://www.mydns.jp
+// T.Kabu <kabu@fvg-on.net> 
```

Comment:
Must be:

```
// Submitted by T.Kabu <kabu@fvg-on.net> 
```

Reply:
OK...
Hummm... I think so it is no good to write a real email address. :-(